src/app/utils/externalLinks.ts

```diff
@@ -48,6 +48,7 @@ export const faucet = {
   [Layer.consensus]: faucetUrl,
   [Layer.emerald]: `${faucetParaTimeBaseUrl}emerald`,
   [Layer.sapphire]: `${faucetParaTimeBaseUrl}sapphire`,
+  [Layer.pontusx]: 'mailto:contact@delta-dao.com?subject=tokens',
```

Comment:
Well... I missed this change 😓

Reply:
And do you have some observation? According to my test, it works great! (I click on the button, MUA opens with email properly prepared.)

Reply:
No, just thought that this would be discussed somewhere, prior to merging(heard it mentioned yesterday at a meeting). Probably it would be simple enough to add PontusX to our faucet, with relatively small changes.

Potentially PontusX could track emails. That was/is my primary concern.

Reply:
So, as far as I can understand, they have already been distributing test tokens this way, requesting them in email, etc. So there is nothing new here, they just continue the same practice on our network.

However, if we can automate it (and make it more private/secure), I'm all for it.